modules/configloader/include/configloader/ProgramOptions.hxx

```diff
@@ -19,23 +19,38 @@
 #ifndef SMARTCAR_EMUL_PROGRAMOPTIONS_HXX
 #define SMARTCAR_EMUL_PROGRAMOPTIONS_HXX
 #include <filesystem>
+#include <boost/predef/os.h>
 
 namespace smce {
 namespace stdfs = std::filesystem;
 
+#if BOOST_OS_UNIX || BOOST_OS_MACOS
+const std::filesystem::path PLATFORM_SMCE_HOME = "/usr/share/smce";
+#elif BOOST_OS_WINDOWS
+const std::filesystem::path PLATFORM_SMCE_HOME = std::filesystem::current_path().root_path() / "Program Files" / "Smce";
+#else
+const std::filesystem::path PLATFORM_SMCE_HOME = ".";
+#endif
```

Comment:
Somewhat anticlimactic to see `std::filesystem` in full letters right after declaring a namespace alias that shortens it